sass/_omni-search.scss

```diff
@@ -23,196 +13,148 @@ $form-padding: 0.5rem;
   padding: $padding;
   }
 
-  ////////////////
-  //// @mga Omni /
-  ////////////////
-@include omnisearch-style(style-1) {
-
-  .nypl-omnisearch {
-    @include omnisearch;
-
-    // @content from the mixin below:
-    button,
-    .nypl-omni-fields,
-    input {
-      display: block;
-      float: left;
-      height: 2rem;
-    }
+@mixin nypl-omnisearch {
+  @include basic-omnisearch(
+    $background-color: $nypl-white,
+    $border: solid 0.0625rem $nypl-dark-gray,
+    $border-radius: $general-border-radius,
+    $padding: 0);
+  font-weight: 200;
 
-    .nypl-omni-fields {
-      width: 20%;
+  @include media($mobile-breakpoint) {
+    width: 97%;
+  }
 
-      label {
-        @include screenreader-only;
-      }
+  button,
+  .nypl-omni-fields,
```

Comment:
Just a code styling comment: maybe keep the basic elements before the classes? So it could be
``` 
button,
input,
.nypl-omni-fields {
```